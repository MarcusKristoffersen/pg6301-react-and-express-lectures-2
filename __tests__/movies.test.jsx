import {ListMovies} from "../index";
import {render} from "react-dom";

describe("movie pages", () => {

    it("shows movies list", () => {
        const element = document.createElement("div");
        render(<ListMovies />, element);
        expect(element.innerHTML).toMatchSnapshot();
    })

    it ("lets the user add a new movie", () => {

    });
});