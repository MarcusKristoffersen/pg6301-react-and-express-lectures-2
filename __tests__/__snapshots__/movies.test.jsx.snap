// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`movie pages shows movies list 1`] = `"<div><h1>Movies</h1><div><h2>Don't look up (2021)</h2><p>Impending disaster, but will politicians act?</p></div><div><h2>Marcus Kristoffersen (2022)</h2><p>Marcus and all them ladies</p></div></div>"`;
